mapping.json

```diff
@@ -41,56 +41,42 @@
     }
   },
   "mappings" : {
-    "group" : {
+    "doc" : {
       "_source" : {
         "enabled" : true
       },
-      "_all" : {
-        "enabled" : true
-      },
       "properties" : {
-        "organizer" : { "type" : "string" },
-        "name" : { "type" : "string" },
+        "type": { "type": "keyword" },
+        "my_join_field": { 
+          "type": "join",
+          "relations": {
+            "group": "event" 
+          }
+        },
+        "organizer" : { "type" : "text" },
+        "name" : { "type" : "text" },
         "description" : {
-          "type" : "string",
+          "type" : "text",
           "term_vector": "with_positions_offsets"
         },
         "created_on" : {
           "type" : "date",
           "format" : "yyyy-MM-dd"
         },
         "tags" : {
-          "type" : "string",
-          "index" : "analyzed",
-          "fields": {  
+          "type" : "text",
+          "index" : "true",
```

Comment:
Can we remove this, since texts are analyzed by default? Or maybe use the boolean `"index": true` instead of the string option? Thanks!